cmd/anonymizer/app/anonymizer/anonymizer.go

```diff
@@ -78,6 +83,8 @@ func New(mappingFile string, options Options, logger *zap.Logger) *Anonymizer {
 			Operations: make(map[string]string),
 		},
 		options: options,
+		ctx:     ctx,
```

Comment:
nit: didn't need to capture ctx in the struct since you're only using it in the same function

Reply:
removed it from the struct